scopesim_templates/metis/laser.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+"""WCU laser spectrum for METIS."""
+
+def laser_spectrum():
+    pass
```

Comment:
```suggestion
    ...
```
:-) I know this is a draft.

I prefer ellipsis for code that isn't complete, because at the same time it looks visually appealing as "something should go here", while at the same time is "bad" (but valid) Python; that is, it is clear that `...` should not stay there. `pass` on the other hand could be intentional and requires more thought to determine whether it needs to be replaced.

This is your draft code, so my thoughts do not matter at all, but maybe you'd like it too.

Reply:
Thx, keep forgetting that `...` is a thing 🙃